### PR TITLE
Implement toolkit python samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/bot-file-upload/python/appManifest/manifest.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "accentColor": "#abcdef",
   "bots": [
     {
-      "botId": "<<MICROSOFT-APP-ID>>",
+      "botId": "${{AAD_APP_CLIENT_ID}}",
```
Comment (frankqianms) [inline samples/bot-file-upload/python/appManifest/manifest.json:28]: Need to change the holder string described in section Run the app (Manually Uploading to Teams) of README.md too.

Please apply this for all 4 samples.

File: samples/bot-file-upload/python/env/.env.local [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# This file includes environment variables that can be committed to git. It's gitignored by default because it represents your local development environment.
+
+# Built-in environment variables
+TEAMSFX_ENV=local
+
+# Generated during provision, you can also add your own variables. If you're adding a secret value, add SECRET_ prefix to the name so Teams Toolkit can handle them properly
+BOT_ENDPOINT=https://fsnhjlfr-3978.inc1.devtunnels.ms
```
Comment (frankqianms) [inline samples/bot-file-upload/python/env/.env.local:7]: need to clear the value here

File: samples/bot-file-upload/python/appManifest/manifest.json (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-file-upload/python/appManifest/manifest.json:28]: I have updated all the placeholders, such as `{{AAD_APP_CLIENT_ID}}`.

**Review by frankqianms — APPROVED** [review]

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/12179155200

**Pull request merged**